pymc_marketing/mmm/utility.py

```diff
@@ -534,12 +534,12 @@ def diversification_ratio(
     The Diversification Ratio is calculated as:
 
     .. math::
-        DR = \frac{\\sum_{i=1}^{n} w_i \\cdot \\sigma_i}{\\sigma_p}
+        DR = \frac{\sum_{i=1}^{n} w_i \cdot \sigma_i}{\sigma_p} \
```

Comment:
According to the Sphinx documentation, the equation doesn't need a newline after the math call:

> When the math is only one line of text, it can also be given as a directive argument:
```.. math:: (a + b)^2 = a^2 + 2ab + b^2```

Also, maybe the line should not terminate with `\` which *might* be the cause of the rendering issue @williambdean is seeing..